core/java/android/ext/settings/ExtSettings.java

```diff
@@ -31,6 +34,12 @@ public class ExtSettings {
     public static final BoolSetting SCRAMBLE_PIN_LAYOUT = new BoolSetting(
             Setting.Scope.PER_USER, "lockscreen_scramble_pin_layout", false);
 
+    public static final IntSetting REMOTE_PROVISIONING_SERVER = new IntSetting(
+        Setting.Scope.GLOBAL, "attest_remote_provisioner_server", // historical setting key
+        GRAPHENEOS_PROXY, // default
+        STANDARD_SERVER, GRAPHENEOS_PROXY, DISABLED // valid values
```

Comment:
`DISABLED` is not a valid value currently.

Reply:
Removed `DISABLED` from valid values